### ISSUE TITLE
Add TFace & Anti-Spoofing models 🎭

### ISSUE THREAD

**Issue by falkaabi** [serengil/deepface#1213]
### Description

Can you please add new SOTA models from this repo: https://github.com/Tencent/TFace

It would also be helpful if you can add new functionality for detecting face spoofing or other face attacks.

All of our community much appreciates your efforts in making this repository the go to repo for face recognition tasks 👍🏻

**Comment by serengil** [comment]
Backbones and pre-trained weights are available at https://github.com/Tencent/TFace/tree/master/recognition

**Comment by Raghucharan16** [comment]
Did you find a way to implement above tencent library models in face recognition.

**Comment by serengil** [comment]
Not yet

**Comment by tanpham380** [comment]
In future will have functionality for detecting face spoofing or other face attacks ? @serengil 
I see https://github.com/Tencent/TFace too interesting

**Comment by serengil** [comment]
Actually I recently added face anti spoofing feature into deepface - https://youtu.be/UiK1aIjOBlQ

But it is not based on TFace

**Comment by tanpham380** [comment]
yeah I have see it in document... but how it can be option like Model for face recognition?

**Comment by serengil** [comment]
@tanpham380 yeah that is why this issue is still open

**Comment by tanpham380** [comment]
:D I like this

**Comment by melffagundes** [comment]
> Hello Sefik, Your deep face library is really great. I am currently using it as a web service. However, I did some tests on Deepface's antispoofing feature, and as a result, it detected some fake photos as real photos. This doesn't give me much confidence because I'm afraid he'll make the same mistake again. Is there a way to fix this permanently?
> 
> Not: I used Deepface.Find function. [@serengil](https://github.com/serengil)

I had the same problem. It detected a photo as a real photo in once on my tests.

@serengil
